lib/util/gitfs/fs/index.ts

```diff
@@ -2,11 +2,25 @@ import * as fs from 'fs-extra';
 import { join, parse } from 'upath';
 import { RenovateConfig } from '../../../config/common';
 import { logger } from '../../../logger';
+import { getChildProcessEnv } from '../../exec/env';
+
+export {
+  pathExists as exists,
+  readFile,
+  writeFile,
+  outputFile,
+  unlink,
+  stat,
+  chmod,
+  move,
+} from 'fs-extra';
```

Comment:
Are re-exports okay or should be avoided?

Reply:
Also would it be good to "gate" all `fs-extra` functions used, not just "when possible"?

Reply:
I think it's ok for us to re-export certain commands. Ideally we have a test that makes sure this file is the only one that imports `fs-extra` or `fs` (or any others applicable).

Reply:
This should be easy doable with a lint rule